### PR TITLE
Completed scan tasks

### PR DESCRIPTION
**Pull request by hinatha**

### CODE REVIEW

File: backend/entities/task.py [hunk 1]
```diff
@@ -17,4 +17,11 @@ def create(body):
...
         }
     table.put_item(Item=item)
-    return json.dumps(item)
\ No newline at end of file
+    return json.dumps(item)
+
+def get():
+    response = table.scan()
+    items = response['Items']
+    return json.dumps(items)
```
Comment (sbk0716) [inline backend/entities/task.py]: https://qiita.com/nagataaaas/items/24e68a9c736aec31948e
https://admin-it.xyz/python/flask-jsonify-jsondumps-difference/
https://qiita.com/nagataaaas/items/24e68a9c736aec31948e

File: frontend/app.py [hunk 2]
```diff
@@ -9,12 +10,16 @@
...
 backend_url = os.getenv('BACKEND_URL', 'http://localhost:5000')
-create_url = f'''{backend_url}/tasks'''
+task_url = f'''{backend_url}/tasks'''
 
 
 @app.route("/")
-def hello():
-    return render_template('index.html')
+def index():
+    r = requests.get(task_url)
```
Comment (sbk0716) [inline frontend/app.py:35]: raise_for_status
https://www.geeksforgeeks.org/response-raise_for_status-python-requests/
https://tonari-it.com/python-response-status-code/

File: frontend/app.py [hunk 3]
```diff
@@ -9,12 +10,16 @@
...
+    l = ast.literal_eval(r.text)
```
Comment (sbk0716) [inline frontend/app.py]: https://note.nkmk.me/python-ast-literal-eval/

File: frontend/app.py [hunk 4]
```diff
@@ -9,12 +10,16 @@
...
+    l_task = [d.get("task") for d in l]
```
Comment (sbk0716) [inline frontend/app.py]: https://www.python.ambitious-engineer.com/archives/154

**Comment by hinatha** [comment]
[#1_UserはTaskの一覧を確認できる](https://trello.com/c/5U9o2Mt3/1-1user%E3%81%AFtask%E3%81%AE%E4%B8%80%E8%A6%A7%E3%82%92%E7%A2%BA%E8%AA%8D%E3%81%A7%E3%81%8D%E3%82%8B)

**Pull request merged**